src/pyOpenMS/tests/unittests/test_Chemistry.py

```
@@ -0,0 +1,570 @@
+
+@report
+def testNASequence():
+    """
+    @tests: NASequence
+     """
+
+    oligo = pyopenms.NASequence.fromString("pAUGUCGCAG");
+
+    assert oligo.size() == 9
+    seq_formula = oligo.getFormula()
+    seq_formula.toString() == u'C86H108N35O64P9'
```

Ruff (0.11.9)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)